app/config/AppConfig.scala

```diff
@@ -104,9 +104,9 @@ class AppConfig @Inject()(
     FeatureStatus.withName(loadConfig(feature2Key(Feature.default)))
 
   def wcoMetadataMapper(): WcoMetadataMapper with WcoMetadataMappingStrategy = {
-      logger.warn("Using WcoMetadataJavaMappingStrategy as the WCO-DEC mapper")
-      new WcoMetadataMapper with WcoMetadataJavaMappingStrategy
-    }
+    logger.warn("Using WcoMetadataJavaMappingStrategy as the WCO-DEC mapper")
```

Comment:
I think it should be debug, it's information for us, not warning. Old implementation, just commented.

Reply:
I have removed it ;)

Reply:
I have removed it